lib/Backend/BailOut.cpp

```diff
@@ -585,7 +585,7 @@ BailOutRecord::RestoreValues(IR::BailOutKind bailOutKind, Js::JavascriptCallStac
         }
 #endif
 
-        if (scriptContext->IsInDebugMode())
+        if (functionBody->GetUtf8SourceInfo()->IsUtf8SourceInfoInDebugMode())
```

Comment:
I would still like to keep it for clarity sake if there are no strong objection.
Makes it explicit when searching for function calls. If all functions are named IsInDebugMode search shows all instance and then need to look at all to find which one applies.